Elevel/Domain/Authorization.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using Elevel.Domain.Models;
+
+namespace Elevel.Domain
+{
+    public class Authorization
+    {
+        public enum Roles
+        {
+            Administrator,
+            HumanResourceManager,
+            Coach,
+            User
+        }
```

Comment:
I would move this enum into a separate file. I think it will be used not only in this file.

Reply:
Done.